src/app/forms/custom-sections/body/body.component.ts

```diff
@@ -104,9 +105,18 @@ export class BodyComponent implements OnInit, OnChanges, OnDestroy {
   }
 
   get bodyTypes(): MultiOptions {
-    const optionsMap = vehicleBodyTypeCodeMap.get(this.techRecord.techRecord_vehicleType as VehicleTypes);
-    const values = [...optionsMap!.values()];
+    let vehicleType: string = this.techRecord.techRecord_vehicleType;
+
+    if (this.techRecord.techRecord_vehicleType === 'hgv' && this.techRecord.techRecord_vehicleConfiguration === 'articulated') {
```

Comment:
Are we going to see more of this type of change in the future for other specific vehicle types ?
if so is there going to be a massive if else chain that can be avoided early ?

Reply:
looking at the ticket, it appears all vehicles have different conditions now for displaying the  https://dvsa.atlassian.net/browse/CB2-9601